_posts/2021-07-08-moveit-galactic.md

```diff
@@ -0,0 +1,31 @@
+---
+author: Henning Kayser
+comments: false
+date: 2021-07-07
+layout: post
+title: MoveIt 2.2 Galactic Release
+media_type: image
+media_link: /assets/images/blog_posts/2021_07_08_galactic.png
```

Comment:
I cannot find this image at /assets/images/blog_posts/

Reply:
ops, added.